test/test_write_then_drop.rs

```diff
@@ -14,9 +14,9 @@ fn write_then_drop() {
 
     let poll = Poll::new().unwrap();
 
-    a.register(&poll, Token(1), Ready::readable(), PollOpt::edge())
+    a.register(&poll, Token(1), Interests::readable(), PollOpt::edge())
         .unwrap();
-    s.register(&poll, Token(3), Ready::empty(), PollOpt::edge())
```

Comment:
Do it matter that the input is changed here @carllerche?

Reply:
I don't think so? I'm not 100% sure what is going on in the original test. Since `Interests` cannot be empty, this change seems fine (if tests pass).